toolchain/ltrace.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2016 IBM.
+# Author: Jayarama<jsomalra@linux.vnet.ibm.com>
+
+# Based on code by
+#   Author: Abdul Haleem <abdhalee@linux.vnet.ibm.com>
```

Comment:
What was the license and copyright of the original code? Where can it be found? Do you have permission from Abdul to re-use and modify it?